website/source/docs/networking/forwarded_ports.html.md

```diff
@@ -111,3 +111,11 @@ The final `:auto_correct` parameter set to true tells Vagrant to auto
 correct any collisions. During a `vagrant up` or `vagrant reload`, Vagrant
 will output information about any collisions detections and auto corrections
 made, so you can take notice and act accordingly.
+
+You can define allowed port range assignable by Vagrant when port collision is
+detected via `config.vm.usable_port_range` property.
+```ruby
```

Comment:
We need a space above this line I think to properly render the code block in Markdown.